### PR TITLE
Feature/r2dbc unit test

### PR DESCRIPTION
**Pull request by santicalleg**
Before submitting a pull request, please read
https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing

## Description
- [Created basic unit tests](https://github.com/bancolombia/scaffold-clean-architecture/commit/7fb1ab7c4d9e26ecfc4ef0cff03da044346593d7)

## Category
- [x] Feature
- [ ] Fix

## Checklist
- [x] The pull request is complete according to the [guide of contributing](https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing)
- [x] Automated tests are written
- [x] The documentation is up-to-date
- [x] The pull request has a descriptive title that describes what has changed, and provides enough context for the changelog
- [ ] If the pull request has changed structural files, you have implemented an UpgradeAction according to the [guide](https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing#upgradeaction)
- [ ] If the pull request has a new Gradle Task, you should add `Analytics` according to the [guide](https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing#analytics)

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,3 +1,3 @@
 package=co.com.bancolombia
-systemProp.version=2.4.9
+systemProp.version=2.4.10-SNAPSHOT
```
Comment (juancgalvis) [inline gradle.properties]: Please rollback this change

**Review by juancgalvis — CHANGES_REQUESTED** [review]
Please resolve the requested change

File: gradle.properties (hunk 1 above)
Comment (santicalleg) [reply gradle.properties]: Hi, ready the rollback, thanks for the feedback

**Review by juancgalvis — APPROVED** [review]
It's ok!

**Pull request merged**